src/eredis_cluster.appup.src

```diff
@@ -1,6 +1,6 @@
 %% -*-: erlang -*-
 
-{"0.6.8",
```

Comment:
missing upgrade instructions for 0.6.8 ?

Reply:
I thought you said to delete the tag so I did. There is no 0.6.8 tag anymore and nothing had pointed to it.